src/Files.App/Views/Settings/AboutPage.xaml

```diff
@@ -3,44 +3,36 @@
 	x:Class="Files.App.Views.Settings.AboutPage"
 	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
 	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
-	xmlns:controls="using:CommunityToolkit.WinUI.UI.Controls"
 	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
 	xmlns:helpers="using:Files.App.Helpers"
-	xmlns:local="using:Files.App.UserControls.Settings"
 	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:settingsuc="using:Files.App.UserControls.Settings"
 	xmlns:vm="using:Files.App.ViewModels.Settings"
+	xmlns:wct="using:CommunityToolkit.WinUI.UI.Controls"
 	mc:Ignorable="d">
 
 	<Page.DataContext>
 		<vm:AboutViewModel x:Name="ViewModel" />
 	</Page.DataContext>
 
-	<Grid>
-		<StackPanel
-			HorizontalAlignment="Stretch"
-			VerticalAlignment="Stretch"
-			Spacing="4">
+	<Grid x:Name="RootGrid">
+		<StackPanel Spacing="{StaticResource SettingsCardSpacing}">
 			<StackPanel.ChildrenTransitions>
-				<TransitionCollection>
-					<RepositionThemeTransition />
-				</TransitionCollection>
+				<EntranceThemeTransition FromVerticalOffset="50" />
```

Comment:
Can you split the unrelated changes into a separate PR?

Reply:
As a general rule of thumb, we try to discuss requirements before starting on a PR. This way we can set the scope and approve any changes before making them.

Reply:
Done.